src/main/app/forms/models/localDate.ts

```diff
@@ -52,7 +52,7 @@ export class LocalDate {
   }
 
   toMoment (): moment.Moment {
-    return moment({ year: this.year, month: this.month - 1, day: this.day }) // Moment months are zero indexed
+    return moment.utc({ year: this.year, month: this.month - 1, day: this.day }) // Moment months are zero indexed
```

Comment:
We've done this before, the fix is to set hours minutes and sections to 0 not using .utc
```
 return moment().hours(0).minutes(0).seconds(0).milliseconds(0)
```
See MomentFactory.currentDate

Reply:
no does't work with this.

Reply:
👍

Reply:
@dharmendrak please confirm that it works still fine when BST timezone is in use. To simulate that please set both your frontend and backend to use BST timezone, submit date of `2018-01-01` and check that the same date is saved in DB.